.github/actions/test-package/action.yml

```diff
@@ -39,8 +39,8 @@ runs:
       DISTRO: ${{ inputs.distro }}
     if: always()
     run: |
-      docker stop k-package-build-${DISTRO} || true
-      docker container rm k-package-build-${DISTRO} || true
+      docker stop -t 0 k-package-build-${DISTRO} || true
+      docker container rm -f k-package-build-${DISTRO} || true
```

Comment:
If options have long and short variants, use the long variants in scripts please (I guess `-t` is `--timeout`?)

Same for `-f`. Make that change throughout the file.

Also, I guess we should remove the ` || true`? If that command doesn't work, something is wrong with our assumptinos about how Docker is working.

Reply:
It would be best to know something fails. Yes removing ` || true` would help us identify an issue sooner rather than later.

Reply:
@ehildenb one thing is with the rm is we can remove this and rely on the automatic clean-up. 

If we wish tokeep in the extra remove call it will require the ||true as it will likely fail more often than not